### PR TITLE
aml-s905x-cc, jetson-nano

### PR DESCRIPTION
**Pull request by abalanonline**
Hello diozero community.

Thank you for the great project. While working on my hobby hardware project, I tried
different java libraries, and diozero performed flawlessly.

Please let me contribute to the project with two board files and one small bugfix.
Boards are Nvidia Jetson Nano and Libre Computer Le Potato.
Files tested on real hardware and worked as expected.

### CODE REVIEW

File: diozero-core/src/main/java/com/diozero/internal/provider/builtin/NativeGpioInputOutputDevice.java [hunk 1]
```diff
@@ -117,16 +117,18 @@ public int getGpio() {
 	public boolean getValue() throws RuntimeIOException {
 		if (mmapGpio == null) {
 			return line.getValue() == 0 ? false : true;
+		} else {
```
Comment (EAGrahamJr) [inline diozero-core/src/main/java/com/diozero/internal/provider/builtin/NativeGpioInputOutputDevice.java:120]: This else is not necessary, as the `return` in the `if` clause will exit the function.

Comment (abalanonline) [reply diozero-core/src/main/java/com/diozero/internal/provider/builtin/NativeGpioInputOutputDevice.java:120]: cosmetic change to make the code of getValue() and setValue() look similar

**Review by mattjlewis — APPROVED** [review]

**Comment by mattjlewis** [comment]
Thank you, always wanted a Jetson device.

**Comment by abalanonline** [comment]
Yes, they are nice. Thank you.

**Pull request merged**